tools/dependencies/known-dependencies.txt

```diff
@@ -431,6 +431,48 @@ azure-storage-file-share-12.16.1.jar
 nimbus-jose-jwt-9.22.jar
 woodstox-core-6.4.0.jar
 azure-core-management-1.9.0.jar
+kyuubi-hive-jdbc-1.7.0.jar
+jcl-over-slf4j-1.7.36.jar
+api-common-2.6.0.jar
+auto-value-1.10.1.jar
+auto-value-annotations-1.10.1.jar
+bcpkix-jdk15on-1.67.jar
+bcprov-jdk15on-1.67.jar
+conscrypt-openjdk-uber-2.5.2.jar
+gapic-google-cloud-storage-v2-2.18.0-alpha.jar
+gax-2.23.0.jar
+gax-grpc-2.23.0.jar
+gax-httpjson-0.108.0.jar
+google-api-client-2.2.0.jar
+google-api-services-storage-v1-rev20220705-2.0.0.jar
+google-auth-library-credentials-1.15.0.jar
+google-auth-library-oauth2-http-1.15.0.jar
+google-cloud-core-2.10.0.jar
+google-cloud-core-grpc-2.10.0.jar
+google-cloud-core-http-2.10.0.jar
+google-cloud-storage-2.18.0.jar
+google-http-client-1.42.3.jar
+google-http-client-apache-v2-1.42.3.jar
+google-http-client-appengine-1.42.3.jar
+google-http-client-gson-1.42.3.jar
+google-http-client-jackson2-1.42.3.jar
+google-oauth-client-1.34.1.jar
+grpc-alts-1.41.0.jar
+grpc-auth-1.41.0.jar
+grpc-google-cloud-storage-v2-2.18.0-alpha.jar
+grpc-googleapis-1.52.1.jar
+grpc-netty-shaded-1.41.0.jar
+grpc-services-1.41.0.jar
+grpc-xds-1.41.0.jar
+opencensus-api-0.31.1.jar
+opencensus-contrib-http-util-0.31.1.jar
+opencensus-proto-0.2.0.jar
+proto-google-cloud-storage-v2-2.18.0-alpha.jar
+proto-google-iam-v1-1.9.0.jar
+re2j-1.6.jar
+threetenbp-1.6.5.jar
+kyuubi-hive-jdbc-1.6.0-incubating.jar
+jcl-over-slf4j-1.7.36.jar
```

Comment:
outdated

Reply:
update kyuubi version, but I keep the others as I'm not sure if these are needed by other people.

Reply:
this PR actually only introduces `kyuubi-hive-jdbc-shaded-1.7.0.jar`